rpc/eth_api.go

```diff
@@ -698,38 +698,36 @@ func (e *PublicEthAPI) GetTransactionReceipt(hash common.Hash) (map[string]inter
 		status = hexutil.Uint(0)
 	}
 
-	res, _, err := e.cliCtx.Query(fmt.Sprintf("custom/%s/%s/%s", types.ModuleName, evm.QueryTxLogs, hash.Hex()))
-	if err != nil {
-		return nil, err
-	}
-
-	var logs types.QueryETHLogs
-	e.cliCtx.Codec.MustUnmarshalJSON(res, &logs)
-
 	txData := tx.TxResult.GetData()
 
 	data, err := types.DecodeResultData(txData)
 	if err != nil {
 		status = 0 // transaction failed
 	}
 
+	// TODO: consider changing to geth's Receipt struct
```

Comment:
geth's GetTransactionReceipt actually returns a map[string]interface{} as well 
https://github.com/ethereum/go-ethereum/blob/b9df7ecdc3d3685180ceb29665bab59e9f614da5/internal/ethapi/api.go#L1337

their Receipt struct doesn't have all the info we want, so I think leaving it like this is fine